packages/ui/PersonalCard/PersonalCard.styles.tsx

```diff
@@ -0,0 +1,97 @@
+import * as HoverCard from "@radix-ui/react-hover-card";
+import * as Popover from "@radix-ui/react-popover";
+import styled, { css } from "styled-components";
+import { Avatar } from "ui/Avatar";
+import { device } from "web/lib/media-queries";
+
+const CommonContentStyled = css`
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  background-color: ${({ theme }) => theme.personalCard.background};
+  border: 1px solid ${({ theme }) => theme.card.border};
+  box-shadow: 4px 4px 8px rgba(0, 0, 0, 0.5);
+  border-radius: 16px;
+  overflow: hidden;
+  gap: 10px;
+  z-index: 11;
+  padding: 0.4em 1em;
+
+  ${device.tablet} {
+    padding: 0.8em 2em;
+  }
+`;
+
+/* HoverCard */
+
+export const WrapperHover = styled.div`
+  display: none;
+  ${device.tablet} {
+    display: inline-block;
+  }
+`;
+
+export const HoverCardContent = styled(HoverCard.Content)`
+  ${CommonContentStyled}
+`;
+
+export const HoverCardArrow = styled(HoverCard.Arrow)`
+  fill: ${({ theme }) => theme.personalCard.main};
+`;
+
+export const UserInfo = styled.div`
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+`;
+
+export const UserName = styled.span`
+  font-size: 16px;
+  ${device.tablet} {
+    font-size: 24px;
+  }
+`;
+
+export const UserEmail = styled.span`
+  font-size: 8px;
+  ${device.tablet} {
+    font-size: 12px;
+  }
+`;
+
+export const AvatarStyled = styled(Avatar)`
+  height: 40px;
+  width: 40px;
+  ${device.tablet} {
+    height: 80px;
+    width: 80px;
+  }
+`;
+
+/* Popover */
+
+export const WrapperPopover = styled.div`
+  display: inline-block;
+  ${device.tablet} {
+    display: none;
+  }
+`;
+
+export const PopoverContent = styled(Popover.Content)`
+  ${CommonContentStyled}
+  flex-direction: column;
+
+  ${device.tablet} {
+    display: none;
+  }
+`;
```

Comment:
```suggestion
export const PopoverContent = styled(Popover.Content)`
  ${CommonContentStyled}
  flex-direction: column;
  max-width: 100vw;

  ${device.tablet} {
    display: none;
  }
`;
```
My suggestion:
![image](https://github.com/intive/patronage2023-js/assets/38572172/a5a6adba-91d5-471e-87b1-c69f96737c6e)

Old one:
![image](https://github.com/intive/patronage2023-js/assets/38572172/b00bffc2-2be3-4c24-b5b3-e3185df92897)

https://github.com/intive/patronage2023-js/blob/2056f416fa542f8d4f16ea4c88701ed951263da5/apps/web/app/(navigation)/budgets/%5Bid%5D/BudgetBasinInformation/PeopleInBudget.tsx#L72

You can change to "top"

https://github.com/intive/patronage2023-js/blob/2056f416fa542f8d4f16ea4c88701ed951263da5/apps/web/components/HiddenUsers/index.tsx#L58

change to "bottom"